qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/search/SearchCustomerInput.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic.portal.portlet.customers.search
+
+/**
+ * Input interface for the {@link life.qbic.portal.portlet.customers.search.SearchCustomer} use case
+ *
+ * This interface describes the methods the use case exposes to its caller.
+ *
+ * @since: 1.0.0
+ *
+ */
+interface SearchCustomerInput {
+
+    void searchCustomer(String firstName, String lastName)
```

Comment:
This is nicely done. Please provide `@since` and a JavaDoc for interface methods.

Reply:
I will look out for this. Thanks for your hint!